jvm-annotations/src/main/java/org/quickperf/jvm/allocation/bytewatcher/ByteWatcherRecorder.java

```diff
@@ -23,29 +23,67 @@
 
     private static final boolean IS_JVM_VERSION_AT_LEAST_12 = JvmVersion.isGreaterThanOrEqualTo12();
 
+    private static JvmVersion jvmVersion = JvmVersion.getCurrentJavaVersion();
+
+    private static int junit4ByteOffset = 40;
+    private static int java12ByteOffsetForJunit4 = 72;
+
+    private static int junit5ByteOffset = 13_728;
+    private static int java9ByteOffsetForJunit5 =  12_120;
+    private static int java10ByteOffsetForJunit5 =  11_744;
+    private static int java12ByteOffsetForJunit5 = 11_296;
+    private static int java13ByteOffsetForJunit5 = 11_264;
+
+    //determine junit4 and junit5 offset statically to avoid as much allocation as possible when calling startRecording
```

Comment:
Could you please remove the word statically

Reply:
done